cc/mcts_player.cc

```diff
@@ -134,11 +137,13 @@ Coord MctsPlayer::SuggestMove() {
   int num_readouts = root_->N() - current_readouts;
   auto elapsed = absl::Now() - start;
   elapsed = elapsed * 100 / num_readouts;
-  std::cerr << "Milliseconds per 100 reads: "
-            << absl::ToInt64Milliseconds(elapsed) << "ms"
-            << " over " << num_readouts
-            << " readouts (batched: " << options_.batch_size << ")"
-            << std::endl;
+  if (options_.verbose) {
+    std::cerr << "Milliseconds per 100 reads: "
+              << absl::ToInt64Milliseconds(elapsed) << "ms"
+              << " over " << num_readouts
+              << " readouts (batched: " << options_.batch_size << ")"
+              << std::endl;
```

Comment:
wanna add "estimated global ms per inference"?

Reply:
I'm going to do that in a different PR when I move the logging over read performance into the DualNet.